### DISCUSSION TITLE
An update for the EU package (+52+021 tile OSM patch)

### DISCUSSION THREAD

**Discussion by mateuszgrzybek** [kubilus1/autoortho#96] [Ideas]
Hey, would it be possible to have the EU ortho set updated? There was an OSM issue with the +52+021 tile (covering a huge part of Warsaw), due to there being a pond defined as a coastline, which would result in Ortho4XP generating the tile as a watery pit. I have patched it a while ago, so the newly generated scenery files should reflect those changes as well.

**Comment by ivotaHR** [comment]
can you share the patch?

**Comment by kubilus1** [comment]
Excellent, let me kick off a rebuild over here: https://github.com/kubilus1/autoortho-scenery/

**Comment by kubilus1** [comment]
Can you point to your patch in the Ortho4XP repo.  I don't see any recent updates, so likely a rebuild wouldn't help.

**Reply by mateuszgrzybek** [reply]
Perhaps I should rephrase my initial post. 
What I meant was a change set within OSM itself, which should now allow the terrain files and coastline definitions to be properly generated by Ortho4XP when it's processing the tile in question.

Please correct me if I'm wrong, but as far as I understand, your project still requires the pre-built '.dsf' and terrain files to fetch the textures on the go?
I thought that simply recreating the said tile (I believe it's in 'z_eur_4') should suffice. If not, then please disregard my request.

**Reply by kubilus1** [reply]
Oh that makes sense.  I'll take a look in a bit.  There is another change that I want to roll in as well.
